### DISCUSSION TITLE
Web3Modal doesn't work on live server but works locally;

### DISCUSSION THREAD

**Discussion by mavenharry1** [WalletConnect/walletconnect-monorepo#2423] [Developer Support] [answered]
I get an error when trying to interact with any contract once I push my code to live server. this happens for all contracts and I see this error message.

<img width="524" alt="Screenshot 2023-05-19 at 1 49 36 PM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/75600248/727ce5c1-db42-4487-9d2b-30ab6869aea9">

**Comment by xzilja** [comment]
This looks like issue related to viem not web3modal. Is your live server running over https?

**Comment by jxom** [comment, marked as answer]
Can you please provide a minimal reproducible example via a separate repository? That will help us debug this issue further. I'm guessing it has to do with your environment/bundler dealing with bigint values.

**Reply by jxom** [reply]
Looks like it might be a `create-react-app` related issue. Happening in MM's signature tools as well... https://github.com/MetaMask/eth-sig-util/issues/304

**Reply by jxom** [reply]
Could be a potential solution for it here: https://github.com/hirosystems/stacks.js/issues/1096#issuecomment-946350299
